src/app/utils/common.util.ts

```diff
@@ -81,10 +81,47 @@ export class CommonUtil {
     return `${toValue.toLocaleString(undefined, { maximumFractionDigits: 2 })}${unit}`;
   }
 
+  static absoluteUsedMemoryColumnFormatter(value: string | undefined): string {
+    if (!value) {
+      return '';
+    }
+    if (value === 'n/a') {
+      return '<strong>Memory:</strong> n/a';
+    }
+    let memory = value.split('%')[0] + '%';
+    return CommonUtil.queueResourceColumnFormatter(memory);
+  }
+
+  static absoluteUsedCPUColumnFormatter(value: string | undefined): string {
+    if (!value) {
+      return '';
+    }
+    if (value === 'n/a') {
+      return '<strong>CPU:</strong> n/a';
+    }
+    let cpu = value.split('%')[1] + '%';
+    cpu = cpu.replace(',', '');
```

Comment:
Maybe we can try catch, so whether or not `value` string in shape, it will return something and not crash.

example: 
```ts
try {
    const parsed = parseResourceValue(value);
    return `<strong>Memory:</strong> ${this.queueResourceColumnFormatter(parsed.memory)}`;
} catch (error) {
    console.error("Error parsing memory value:", error);
    return '<strong>Memory:</strong> Error';
}
```